packages/ansi/src/sixel/decoder.ts

```
@@ -0,0 +1,496 @@
+    height: number;
+    palette: SixelPalette;
+    pixelAspectRatioDenominator: number;
+    pixelAspectRatioNumerator: number;
+    pixels: Uint8Array; // Pixel data, where each byte is an index into the palette
+    width: number;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+export interface SixelDecoderOptions {
+    // Placeholder for future options, e.g., custom palette, error handling strategy
+}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)